src/demo/components/login/login.component.html

```diff
@@ -90,6 +90,19 @@ <h1>Demo</h1>
             Customer not found
           </mat-error>
         </mat-form-field>
+
+        <mat-form-field appearance="outline">
```

Comment:
Do we need to set the bank guid if the customer guid has to be set? We'd need to validate the bank guid against the customer.

Reply:
Now that the customer object includes bank ownership we no longer need to validate for this.